include/klee/util/Bits.h

```diff
@@ -88,9 +95,11 @@ namespace klee {
     // @pre((x&(x-1)) == 0)
     // @post((1 << retval) == x)
     inline unsigned indexOfSingleBit(uint64_t x) {
+      assert((x & (x - 1)) == 0);
       unsigned res = bits32::indexOfSingleBit((unsigned) (x | (x>>32)));
-      if (x&((uint64_t) 0xFFFFFFFF << 32))
-	  res += 32;
+      if (x & ((uint64_t)0xFFFFFFFF << 32))
```

Comment:
Both points are very valid. These days it's difficult to get changes accepted into KLEE so I kind of feel forced into throwing as many changes into a PR as possible even though they might not be related.

If you want to do it feel free to add an additional commit that switches to using the `UINT64_C()` macro.